src/person.rb

```diff
@@ -1,5 +1,7 @@
+require './nameable'
+
 class Person
```

Comment:
- [  OPTIONAL ] Hey there! 🚀 I was thinking, wouldn't it be awesome if we moved those getters and setters right up to the top of the class? 📦🚀 What do you reckon? 🤔 Let's aim to declare the getters and setters at the top of the class for maximum clarity and organization! 🌟👩‍💻🌈

Reply:
Alright. Thank you for this suggestion. I would implement it.